packages/app-store/dailyvideo/lib/VideoApiAdapter.ts

```diff
@@ -130,6 +137,8 @@ const DailyVideoApiAdapter = (): VideoApiAdapter => {
           enable_chat: true,
           exp: exp,
           enable_recording: "cloud",
+          enable_transcription: `deepgram:${process.env.DEEPGRAM_API_KEY}`,
+          permissions: { canAdmin: ["transcription"] },
```

Comment:
we should also add it to https://github.com/calcom/cal.com/blob/main/apps/ai/.env.example